_examples/docscansandbox/demo_test.go

```diff
@@ -0,0 +1,338 @@
+	}
+
+	parsedURL, err := url.Parse(iFrameURL)
+	if err != nil {
+		panic(err)
+	}
+
+	query, err := url.ParseQuery(parsedURL.RawQuery)
+	if err != nil {
+		panic(err)
+	}
+
+	return query.Get("sessionID")
+}
+
+func (c *webContext) iConfigureTheSessionResponse() error {
+	documentFilter, err := filter.NewDocumentFilterBuilder().
+		WithDocumentType("PASSPORT").
+		Build()
+	if err != nil {
+		return err
+	}
+
+	authenticityBreakdown, err := report.NewBreakdownBuilder().
+		WithSubCheck("security_features").
+		WithResult("NOT_AVAILABLE").
+		WithDetail("some_detail", "some_detail_value").
+		Build()
+	if err != nil {
+		return err
+	}
+
+	authenticityRecommendation, err := report.NewRecommendationBuilder().
+		WithValue("NOT_AVAILABLE").
+		WithReason("PICTURE_TOO_DARK").
+		WithRecoverySuggestion("BETTER_LIGHTING").
+		Build()
+	if err != nil {
+		return err
+	}
+
+	documentAuthenticityCheck, err := check.NewDocumentAuthenticityCheckBuilder().
+		WithBreakdown(authenticityBreakdown).
+		WithRecommendation(authenticityRecommendation).
+		WithDocumentFilter(documentFilter).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textDataCheckBreakdown, err := report.NewBreakdownBuilder().
+		WithSubCheck("document_in_date").
+		WithResult("PASS").
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textDataCheckRecommendation, err := report.NewRecommendationBuilder().
+		WithValue("APPROVE").
+		Build()
+
+	textDataCheck, err := check.NewDocumentTextDataCheckBuilder().
+		WithBreakdown(textDataCheckBreakdown).
+		WithRecommendation(textDataCheckRecommendation).
+		WithDocumentFields(map[string]string{
+			"full_name":       "John Doe",
+			"nationality":     "GBR",
+			"date_of_birth":   "1986-06-01",
+			"document_number": "123456789",
+		}).
+		WithDocumentFilter(documentFilter).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	checkReports, err := request.NewCheckReportsBuilder().
+		WithAsyncReportDelay(5).
+		WithDocumentAuthenticityCheck(documentAuthenticityCheck).
+		WithDocumentTextDataCheck(textDataCheck).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textExtractionTask, err := task.NewDocumentTextDataExtractionTaskBuilder().
+		WithDocumentFields(map[string]string{
+			"full_name":       "John Doe",
+			"nationality":     "GBR",
+			"date_of_birth":   "1986-06-01",
+			"document_number": "123456789",
+		}).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	taskResults, err := request.NewTaskResultsBuilder().
+		WithDocumentTextDataExtractionTask(textExtractionTask).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	responseConfig, err := request.NewResponseConfigBuilder().
+		WithCheckReports(checkReports).
+		WithTaskResults(taskResults).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	configErr := c.client.ConfigureSessionResponse(c.getIFrameSessionID(), responseConfig)
+
+	if configErr != nil {
+		request, _ := json.Marshal(responseConfig)
+		return errors.New(string(request) + configErr.Error())
+	}
+
+	return configErr
```

Comment:
configErr will always be nil here, so might be clearer what we're returning by just returning nil here? 
```suggestion
	return nil
```